Dangerfile

```diff
@@ -1,6 +1,6 @@
 #### HELPER METHODS
 def fail_if_no_supported_label_found
-  supported_types = ["breaking", "build", "ci", "docs", "feat", "fix", "perf", "refactor", "style", "test", "next_release", "dependencies", "phc_dependencies"]
+  supported_types = ["breaking", "build", "ci", "docs", "feat", "fix", "perf", "refactor", "RevenueCatUI", "style", "test", "next_release", "dependencies", "phc_dependencies"]
```

Comment:
I made a change in https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal/pull/54 so its case insensitive, but I think this probably has to match the case.